### PR TITLE
Authorization using (comments)

### PR DESCRIPTION
**Pull request by un-tone**
For https://github.com/StarcounterApps/Images/issues/113.

Please feel free to ask and add notes.

### CODE REVIEW

File: src/Images/Api/AuthorizedHandlers.cs [hunk 1]
```diff
@@ -12,64 +12,110 @@
 
 namespace Images
 {
+    /// <summary>
+    /// Registering handlers and permission rules using Authorization library.
+    /// </summary>
+    /// <remarks>
+    /// In the Images app we don't use UrlAttribute provided by the Authorization library.
```
Comment (PatrykSzwer) [inline src/Images/Api/AuthorizedHandlers.cs]: I understand that we dont use UrlAttribute here, but would be nice If You could add short explanation here **why** We're not using just the included UrlAttribute. Is It to separate partial and public URIs? If It is so, then why we have to do it? And Should We do same in other apps?

Comment (PatrykSzwer) [inline src/Images/Api/AuthorizedHandlers.cs]: I understand that we dont use UrlAttribute here, but would be nice If You could add short explanation here **why** We're not using just the included UrlAttribute. Is It to separate partial and public URIs? If It is so, then why we have to do it? And Should We do same in other apps?

File: src/Images/Api/AuthorizedHandlers.cs [hunk 2]
```diff
@@ -12,64 +12,110 @@
...
                 .GetTypes()
                 .Select(type => Tuple.Create(type, type.GetCustomAttribute<PartialUrlAttribute>()))
                 .Where(tuple => tuple.Item2 != null)
                 .ToDictionary(x => x.Item1, x => x.Item2);
 
+            // create two Router instances - one for public-accessible URIs and another one for partial (self-only) URIs
             Router outsideFacingRouter = CreateApiRouter();
             Router selfOnlyRouter = CreatePartialsRouter();
 
+            // go through the pages marked with PartialUrl attribute and define handlers for its public and partial URIs
```
Comment (PatrykSzwer) [inline src/Images/Api/AuthorizedHandlers.cs]: Could You please explain to me that step a little bit further? Why We have to do this? Maybe not necessarily in code, just here in a comment.

File: src/Images/Api/AuthorizedHandlers.cs (hunk 1 above)
Comment (un-tone) [inline src/Images/Api/AuthorizedHandlers.cs]: Tried to answer. Please check.

File: src/Images/Api/AuthorizedHandlers.cs (hunk 2 above)
Comment (un-tone) [inline src/Images/Api/AuthorizedHandlers.cs]: Tried to answer. Please check.

**Comment by warpech** [comment]
@habib535, @PatrykSzwer can this be merged now or are there any more questions?

**Pull request merged**